sourcegraph/sourcegraph
# Add local storage persist layout for code insight dashboards

Partially closes https://github.com/sourcegraph/sourcegraph/issues/29960

## Test plan
- Go to any code insights dashboard
- Change size and order 
- Refresh the page or go away from this dashboard and go back to it 
- Check that insights are in the same places where you put them last time
- Add new insights and remove some insights from the dashboard
- Grid system shouldn't be broken after all actions (add/remove)
